File: app/src/main/java/com/angogasapps/myfamily/utils/FamilyManager.java
package com.angogasapps.myfamily.utils;

import android.net.Uri;

import com.angogasapps.myfamily.firebase.dynamic_links.DynamicLinksManager;
import static com.angogasapps.myfamily.firebase.FirebaseVarsAndConsts.USER;

import java.util.HashMap;

public class FamilyManager {
    public static String PARAM_FAMILY_ID = "family";

    public static Uri getInviteLinkToFamily(){
        
        HashMap<String, String> params = new HashMap<>();
        params.put(PARAM_FAMILY_ID, USER.getFamily());

        return DynamicLinksManager.createDynamicLinkWithParams(params);
    }
}

File: app/src/main/java/com/angogasapps/myfamily/utils/FamilyManager.kt
package com.angogasapps.myfamily.utils

import android.net.Uri
import com.angogasapps.myfamily.firebase.USER
import com.angogasapps.myfamily.firebase.dynamic_links.DynamicLinksManager.createDynamicLinkWithParams

import java.util.HashMap

object FamilyManager {
    var PARAM_FAMILY_ID = "family"
    val inviteLinkToFamily: Uri
        get() {
            val params = HashMap<String?, String?>()
            params[PARAM_FAMILY_ID] = USER.family
            return createDynamicLinkWithParams(params)
        }
}